packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -6460,8 +6460,18 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
       Kokkos::create_mirror_view(this->mj_coordinates);
     Kokkos::deep_copy(host_src_coordinates, this->mj_coordinates);
     for(int i = 0; i < this->coord_dim; ++i) {
-      auto sub_host_src_coordinates
-        = Kokkos::subview(host_src_coordinates, Kokkos::ALL, i);
+      Kokkos::View<mj_scalar_t*, Kokkos::LayoutLeft, device_t>
+        sub_host_src_coordinates;
+      if(host_src_coordinates.extent(0) == 0) {
+        // view could be size 0 if graph was not distributed - make an empty view
+        sub_host_src_coordinates =
```

Comment:
You don't need this; it's already an empty View (that's what the default constructor does).  
```c++
if (host_src_coordinates.extent(0) != 0) {
  sub_host_src_coordinates = 
    Kokkos::subview(host_src_coordinates, Kokkos::ALL, i);
}
```

Reply:
Thanks. Resolved.